demo/addons/isochroneAddon/isochroneAddon.html

```diff
@@ -0,0 +1,75 @@
+﻿<div id="custom-popin" style="display:none;">
+      <div class="header">
+        <div id="header-content">
+          <i class="fas fa-lg fa-grip-vertical"></i><h3 id="addon_title"> </h3>
+        </div>
+      </div>
+      <div class="body">
+        <div>
+            <div id="depart">
+              <h4><i class="fas fa-thumbtack"></i> Point d'origine</h4>
+              <button type="button" id="choisir_depart" class=" btn btn-default"><i class="fas fa-crosshairs"></i> Choisir</button>
+            </div>
+
+            <div id="parametre-container">
+              <h4>Paramètres</h4>
```

Comment:
Could use I18n to.

Reply:
It would be nice but not required for demo contrib.